AIMS/Views/Home/HomePageCardComponent.cshtml

```diff
@@ -1,37 +1,86 @@
-<div class="homepage-cards">
-    <div class="card">
-    <h3>Monitors</h3>
-    <p class="total">150 total</p>
-    <p class="available">Available: 120</p>
-</div>
-
-<div class="card">
-    <h3>Laptops</h3>
-    <p class="total">120 total</p>
-    <p class="available">Available: 85</p>
-</div>
-
-<div class = "card">
-    <h3>Desktops</h3>
-    <p class="total">100 total</p>
-    <p class="available">Available: 70</p>
-</div>
-
-<div class="card">
-    <h3>Software Licenses</h3>
-    <p class="total">200 total</p>
-    <p class="available">Available: 180</p>
-</div>
-
-<div class="card">
-    <h3>Headsets</h3>
-    <p class="total">80 total</p>
-    <p class="available">Available: 60</p>
-</div>
-
-<div class="card">
-    <h3>Charging Cables</h3>
-    <p class="total">90 total</p>
-    <p class="available">Available: 75</p>
+@{
+    Layout = null;  
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Card Component Preview</title>
+
+    <!-- Link to CSS -->
+    <link rel="stylesheet" href="~/css/HomeCardComponent.css" asp-append-version="true">
+    <style>
+        .card-link {
+            text-decoration: none;
+            color: inherit;
+        }
+    </style>
+</head>
+<body>
+
+    <div id="card-container">
+        <a href="/Home/AssetDetailsComponent?category=Monitor" class="card-link">
```

Comment:
For creating links like these, I think we should prefer to use the dynamic generation provided by Razor: https://www.codingfusion.com/Post/-Net-MVC-Html-ActionLink-tutorial-with-examples.